docs/src/lib/utilities.md

```diff
@@ -11,8 +11,8 @@ Pages = ["utilities.md"]
 ```@docs
 eltypes
 head
-complete_cases
```

Comment:
I don't know what this file is for. Is this used to generate online docs? It's an incomplete list of functions available in DataTables and I wasn't sure what to do with it